stable/elasticsearch-curator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "5.7.6"
 description: A Helm chart for Elasticsearch Curator
 name: elasticsearch-curator
-version: 2.1.5
+version: 2.1.6
```

Comment:
This is new feature according to semver, could you bump minor version instead of patch?

Reply:
only this change request and we can merge it

Reply:
done